thirdparty/rabbitmq-server/zimbra-rabbitmq-server/debian/rules

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/make -f
+export DEB_BUILD_OPTIONS=nocheck
+
+unexport DEPS_DIR
+
+# Ensure rpath is set correctly
+export DEB_LDFLAGS_MAINT_APPEND=-Wl,-rpath,OZ
+
+%:
+	dh $@
+
+
+override_dh_shlibdeps:
+	dh_shlibdeps --dpkg-shlibdeps-params=--ignore-missing-info
+
+override_dh_auto_build:
+	make dist manpages
+
+override_dh_strip:
+	dh_strip -pzimbra-rabbitmq-server
+	dh_strip -Nzimbra-rabbitmq-server
+
+
+override_dh_auto_install:
+	make install install-bin install-man DESTDIR=$$(pwd)/debian/tmp PREFIX=/opt/zimbra RMQ_ROOTDIR="/opt/zimbra/common/lib/rabbitmq" RMQ_ERLAPP_DIR=/opt/zimbra/common/lib/rabbitmq/lib/rabbitmq_server-3.10.7 MANDIR="/opt/zimbra/common/share/man"
+	rm -rf $$(pwd)/debian/tmp/opt/zimbra/common/lib/rabbitmq/lib/rabbitmq_server-3.10.7/LICENSE*
```

Comment:
when we upgrade rabbitmq in future we will have to update these paths as it contains version, is there a way to prevent that situation?

Reply:
updated changes